src/components/feedbackpage/FeedbackPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { useState } from 'react';
```

Comment:
Since you are already importing something from `react` in `line #1` why don't you add this import there itself